### DISCUSSION TITLE
How to enable alternative cut/copy/paste shortcuts with "insert" button?

### DISCUSSION THREAD

**Discussion by slashsbin** [Manjaro-Sway/manjaro-sway#586] [Q&A]
I would like to enable these keyboard shortcuts:

- Copy `Ctrl+Insert`
- Past `Shift+Insert`
- Cut `Shift+Delete`

It looks like that, it sometimes work with some applications and not with some other.

**Comment by boredland** [comment]
Can you share how you set them in your sway config?

**Comment by slashsbin** [comment]
I didn't. I'm looking for the way of how to set them.
